src/bundler.js

```diff
@@ -361,32 +367,34 @@ function processJsBundle(options, jsBundle, bundleDir, jsFiles, bundleName, cb)
                             bundleStatsCollector.ParseJsForStats(jsBundle, code);
                         }
 
-                        next(code);
+                        next({
+                            code: code
+                        });
 
                     }
 
 
                 });
 
-                if (options.outputbundlestats) {
-                    bundleStatsCollector.AddDebugFile(jsBundle, jsPath);

```

Comment:
This was a bug introduced in a prior PR of mine. This resulted in the map of bundle to debug files containing only original, uncompiled files rather than the paths to compiled files.